.github/workflows/simd-tier-correctness.yml

```
@@ -1,122 +1,122 @@
-        shell: pwsh
-        run: cmake --preset msvc-debug -DDMK_ENABLE_AVX512=ON
-
-      - name: Build tests
-        shell: pwsh
-        run: cmake --build build/msvc-debug --target DetourModKit_tests --parallel 3
-
-      # Intel SDE executes the selected tier on any CPU, so it validates each tier's results without that hardware.
-      # The SHA-pinned setup-sde action exposes the kit in SDE_PATH. A setup failure fails the leg: without the
-      # emulator this job has no way to reach the tier it is named for.
-      - name: Set up Intel SDE
+name: Scanner SIMD tier correctness
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)